it/it-tests/src/test/java/it/Category2Suite.java

```diff
@@ -67,7 +69,9 @@
   IssueActionTest.class,
   IssueBulkChangeTest.class,
   IssueChangelogTest.class,
+  IssueCreationTest.class,
```

Comment:
As I wanted to add a new IT, I've added in IssueMeasureTest, but it was failing => As jobs were green I suspect that it was not executed.
Yes, since we've re-organized the ITs it's now easy to forget test, but there's no easy solution for the moment.